packages/terminal/src/browser/terminal-theme-service.ts

```diff
@@ -177,7 +177,7 @@ export class TerminalThemeService {
         };
         // eslint-disable-next-line guard-for-in
         for (const id in terminalAnsiColorMap) {
-            const colorName = id.substring(13);
+            const colorName = id.substring(13).toLowerCase();
```

Comment:
I don't think it is correct, it will translate `ansiBrightWhite` to `brightwhite` when a proper one is `brightWhite`. Only first character should be converted to lower case.